src/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
- [ ] Kindly follow the requirement given by the previous reviewer, I can see that you did not really understand, what was the requested change, what you need to do is that: Create a folder into your `scr ` foder called `redux` then create two other folders (`books and categories`) inside the redux folder after that you can now create one file inside those two folders, the first one will be `books.js` inside the book folder and the second one `categories.js` inside the categories folder
- see how the tree will look like:

``` ...
/src
    |---/components
    |---/redux
        |--- /books
                | books.js
        |--- /categories
                | categories.js
        | configureStore.js
    | index.js
```